### PR TITLE
New playtest page

### PR DESCRIPTION
**Pull request by PunkPun**
WIP

### CODE REVIEW

**Comment by Orb370** [comment]
12: "exited" -> excited (first paragraph)

32: Our netcode has had some major improvements in both low-latency and stalling games
^Needs a period

43: "Polish changes" isn't usually a thing people say. I would reword this sentence. 

54:  replace directly with more

57: remove "inside"

71: Change to: Many hotkeys are now rebindable

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    <img src="{{ '/images/news/20221031-transient.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Nuke incomming!</figcaption>
+</figure>
+
+One of the highlight features is the new hierarchical pathfinder. It should increase performance massively on large or more complex maps.
+
+<figure>
+  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obsticles ahead of them" />
+  <figcaption>HPF helps units better predict obsticles ahead of them.</figcaption>
```
Comment (Mailaender) [inline news/_posts/2022-10-30-playtest-20221030.md:29]:
```suggestion
  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />
  <figcaption>HPF helps units better predict obstacles ahead of them.</figcaption>
```

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+ * Loaders for more audio formats (e.g. mp3, ogg), and formats used by the remasters
+ * Many linting and documentation updates
+
+As well as some other changes worth mentioning:
+ * Toolchain and runtime changes to .NET 6
+ * A massive amount of upgrades to both ingame and client UI
+ * Many more hotkeys are now rebindable
+ * Substantial progress towards localisation
+
+This post has covered just a small subset of the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)), and we encourage anybody interested in finding out more to read the full list. Head on over to the [download page]({{ '/download' | relative_url }}) to install the new playtest, and let us know what you think in the comments below, on [our forum](https://forum.openra.net/), [community Discord server](https://discord.gg/UcUztfr), or [GitHub](https://github.com/OpenRA/OpenRA/issues).
```
Comment (Mailaender) [inline news/_posts/2022-10-30-playtest-20221030.md:91]: Next stable is https://github.com/OpenRA/OpenRA/wiki/Changelog not bleed.

Comment (Mailaender) [reply news/_posts/2022-10-30-playtest-20221030.md:91]: community Discord server is https://discord.openra.net

Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:91]: > Next stable is https://github.com/OpenRA/OpenRA/wiki/Changelog not bleed.

Yes, I'll update it once we have that changelog

> community Discord server is https://discord.openra.net/

Good catch, I just copied it over from another page

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+ * New Red Alert missions
+ * An ingame encyclopedia in Dune 2000
+ * Measures against spamming in-game chat
+ * The ability to mute players
+ * Support for disabling the chat in replays
+
+Notable changes for mappers and modders include:
+ * Automatic map reloading in the background
+ * Changed the way speed is calculated for ground units
+ * New map format which includes the map preview in the map versioning (map hash)
```
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:81]: It's for mappers and modders, it's fine if it's a bit more technical

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Notable changes for mappers and modders include:
+ * Automatic map reloading in the background
+ * Changed the way speed is calculated for ground units
+ * A new map format which includes the map preview (minimap) in the map versioning (map hash)
+ * Loaders for more audio formats (e.g. mp3, ogg), and formats used by the remasters
+ * Many linting and documentation updates
+
+As well as some other changes worth mentioning:
+ * Toolchain and runtime changes to .NET 6
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:82]:
```suggestion
 * Toolchain changes to support the .NET 6 runtime
```
(nts: revisit?)

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+Some of the more visible new features and improvements include:
+ * Measures against spamming in-game chat
+ * The ability to mute players
+ * Support for disabling the chat in replays
+ * New Red Alert missions
+ * An ingame encyclopedia in Dune 2000
+
+Notable changes for mappers and modders include:
+ * Automatic map reloading in the background
+ * Changed the way speed is calculated for ground units
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:76]:
```suggestion
 * A change to the way speed is calculated for ground units
```
(nts: revisit)

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-td.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-td.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-td.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Forces collide</figcaption>
+</figure>
+
+We've added transient text notifications matching speech notifications. Now you can play without sound!
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:32]:
```suggestion
We added transient text notifications to match speech notifications. Now you can play without sound!
```

We could include something here along the lines of "Now it is harder than ever to miss important notifications during battle" to give this a spin that it doesn't just benefit people who play without sound. (nts: revisit)

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+Our netcode has had some major improvements in both low and high latency games.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-td.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-td.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-td.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Forces collide</figcaption>
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:29]:
```suggestion
  <figcaption>Forces collide.</figcaption>
```

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+Tiberian Sun has also received its fair share of features and bug fixes. This time around the highlight is tilting on slopes.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-slope.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-slope.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-slope.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>OpenRA now supports tilting on slopes!</figcaption>
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:62]:
```suggestion
  <figcaption>OpenRA now supports tilting on slopes.</figcaption>
```

**Review by abcdefg30 — COMMENTED** [review]
Preliminary review

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
+---
+kind: article
+title: "Playtest 20221030"
+author: "PunkPun, Therapist"
+created_at: 2022-10-30 22:00 +0000
+disqus_id: "playtest-20221030"
+permalink: "/news/playtest-20221030/"
+---
+
+Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!
```
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:10]: This really kills the energy

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+
+There's been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we're excited to finally showcase a number of new features that've been built up over the years!
+
+One of the highlight features is the new hierarchical pathfinder. It should increase performance massively on large or more complex maps.
+
+<figure>
+  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />
+  <figcaption>HPF helps units better predict obstacles ahead of them.</figcaption>
```
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:18]: The sentence doesn't feel whole with this change

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+created_at: 2022-10-30 22:00 +0000
+disqus_id: "playtest-20221030"
+permalink: "/news/playtest-20221030/"
+---
+
+Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!
+
+There's been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we're excited to finally showcase a number of new features that've been built up over the years!
+
+One of the highlight features is the new hierarchical pathfinder. It should increase performance massively on large or more complex maps.
```
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md]: I'd rather have complex (not claustrophobic. Implemented other suggestion

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+ * Automatic map reloading in the background
+ * Changed the way speed is calculated for ground units
+ * A new map format which includes the map preview (minimap) in the map versioning (map hash)
+ * Loaders for more audio formats (e.g. mp3, ogg), and formats used by the remasters
+ * Many linting and documentation updates
+
+As well as some other changes worth mentioning:
+ * Toolchain and runtime changes to .NET 6
+ * A massive amount of upgrades to both ingame and client UI
+ * Many more hotkeys are now rebindable
```
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:84]: We didn't really add hotkeys, just made them rebindable

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+There's been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we're excited to finally showcase a number of new features that've been built up over the years!
+
+One of the highlight features is the new hierarchical pathfinder. It should increase performance massively on large or more complex maps.
+
+<figure>
+  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />
+  <figcaption>HPF helps units better predict obstacles ahead of them.</figcaption>
+</figure>
+
+Our netcode has had some major improvements in both low and high latency games.
```
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:21]: This kills the energy

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+title: "Playtest 20221030"
+author: "PunkPun, Therapist"
+created_at: 2022-10-30 22:00 +0000
+disqus_id: "playtest-20221030"
+permalink: "/news/playtest-20221030/"
+---
+
+Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!
+
+There's been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we're excited to finally showcase a number of new features that've been built up over the years!
```
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md]: Honestly I dunno how best to rewrite it. I've implemented the second part of the suggestionI don't know how to fix the first part

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 6 above)
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:32]: Perhaps

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+
+The fairly recent Dune film has brought in many players into Dune 2k and a decently sized multiplayer community was born. With the feedback from the community Dune 2k has received a number of balance and polish changes.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-dune.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-dune.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-dune.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Storm is approaching!</figcaption>
```
[the remark this answers is not in the document]
Comment (abcdefg30) [reply news/_posts/2022-10-30-playtest-20221030.md:51]: That reads weird though, Imo.

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (abcdefg30) [reply news/_posts/2022-10-30-playtest-20221030.md:21]: I'm not sure what you mean by that. If anything this increases the energy by using active voice? (And fits the rest of the article better.)

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (abcdefg30) [reply news/_posts/2022-10-30-playtest-20221030.md:18]: The sentence isn't really whole (i.e. correct English) before this change. If you want to keep `better` then it should read
```suggestion
  <figcaption>HPF helps units to better predict obstacles ahead of them.</figcaption>
```

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (abcdefg30) [reply news/_posts/2022-10-30-playtest-20221030.md:10]: Not sure what you mean, we still keep one `!`. Not having the comma and keeping two exclamation marks is bad style.

[the remark this answers is not in the document]
Comment (abcdefg30) [reply news/_posts/2022-10-30-playtest-20221030.md:10]: (If you insist we can keep the second `!`, but I'd still add the comma.)

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+
+Notable changes for mappers and modders include:
+ * Automatic map reloading in the background
+ * A change to the way speed is calculated for ground units
+ * A new map format which includes the map preview (minimap) in the map versioning (map hash)
+ * Loaders for more audio formats (e.g., mp3 and ogg), and formats used by the remasters
+ * Many linting and documentation updates
+
+As well as some other changes worth mentioning:
+ * Toolchain and changes to support .NET 6 runtime
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:82]: You missed the article
```suggestion
 * Toolchain and changes to support the .NET 6 runtime
```

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:10]: I'm mostly talking about the comma. Adding it literally kills the tempo

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:84]: changed the time

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:21]: The way you phrase it makes it sound like I need a second sentence or image to follow it up with. But really I want this to be a standalone statement

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (abcdefg30) [reply news/_posts/2022-10-30-playtest-20221030.md:10]: Apparently this isn't just bad style but plain incorrect:
> You need a comma after “welcome” if it is an introductory interjection or if it is immediately followed by the name or another word directly addressing the people being spoken to.

Whether or not it kills tempo is subjective, and the EVA announcement also has it if you look at the end of the video: https://www.youtube.com/watch?v=JkWkYZgLk2I

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:51]: removed it

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (abcdefg30) [reply news/_posts/2022-10-30-playtest-20221030.md:21]: Yes, it probably should. Having a single disconnected sentence feels a bit weird to me.

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:10]: sure...

**Comment by abcdefg30** [comment]
The TS gif shows a lot of clipping with the Titan. Can we use a different gif?

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+There have been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we are excited to finally showcase a number of new features and improvements that have been built up over the last year and a half.
+
+One highlight feature is the new hierarchical pathfinder, which massively increases performance on large or more complex maps.
+
+<figure>
+  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />
+  <figcaption>HPF helps units to better predict obstacles ahead of them.</figcaption>
+</figure>
+
+Our networking code has had some major improvements in both low and high latency games.
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:21]: Maybe
```suggestion
Our new netcode brings major improvements in both low and high latency games. Clients that experience connection hiccups can now catch up and do not slow the game permanently.
```
if you want more than one sentence. (It still irks me a bit that this reads like the netcode has improvements during games... while it is the connection that sees those improvements.)

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 18]
```diff
@@ -0,0 +1,87 @@
+---
+kind: article
+title: "Playtest 20221030"
+author: "PunkPun"
+created_at: 2022-10-30 22:00 +0000
+disqus_id: "playtest-20221030"
+permalink: "/news/playtest-20221030/"
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:7]: Dates need to be updated.

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+---
+
+Welcome back, Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It is finally here!
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:10]: Same here and for the image names.

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+
+As well as some other changes worth mentioning:
+ * Toolchain and changes to support the .NET 6 runtime
+ * A massive amount of upgrades to both ingame and client UI
+ * Many more hotkeys became rebindable
+ * Substantial progress towards localisation
+
+All in all, 615 pull requests were merged since last release and 552 issues were closed bringing the total number of open tickets down by 141.
+
+This post covered just a small subset of all the changes and improvements added since the last release. We encourage anybody interested in finding out more to read the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog). Head on over to the [download page]({{ '/download' | relative_url }}) to install the new playtest and let us know what you think in the comments below, on [our forum](https://forum.openra.net/), [community Discord server](https://discord.openra.net) or on [GitHub](https://github.com/OpenRA/OpenRA/issues). Happy playtesting!
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:87]: https://github.com/OpenRA/OpenRA/wiki/Changelog/1e2499c01b75fc76018dfe96ce915379557b3716 is the full changelog link.

File: news/_posts/2022-10-30-playtest-20221030.md [hunk 21]
```diff
@@ -0,0 +1,87 @@
...
+    <source src="{{ '/images/news/20221031-td.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-td.png' | relative_url }}" width="700" />
+  </video>
+</figure>
+
+We have added transient text notifications matching speech notifications. Now you can play without sound!
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-transient.mp4' | relative_url }}" type="video/mp4">
```
Comment (abcdefg30) [inline news/_posts/2022-10-30-playtest-20221030.md:35]:
```suggestion
    <source src="{{ '/images/news/20221031-transient.mov' | relative_url }}" type="video/mp4">
```
?

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 17 above)
Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:21]: it's not just the connection though. The way we handle the delay is different

Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md:21]: it's not just about the hiccups, the whole game just plainly runs smoother and is more responsive

**Comment by PunkPun** [comment]
> The TS gif shows a lot of clipping with the Titan. Can we use a different gif?

If you can make it. The titan is pretty glitchy yes, but It's not jarring and I think it's fine as it clearly shows that TS is not ready. While trying to get a good video I encountered way worse visual gliches

File: news/_posts/2022-10-30-playtest-20221030.md (hunk 17 above)
Comment (abcdefg30) [reply news/_posts/2022-10-30-playtest-20221030.md:21]: Then mention this too. How is an end-user supposed to know this from the single sentence?

Comment (PunkPun) [reply news/_posts/2022-10-30-playtest-20221030.md]: Sure

File: news/_posts/2022-12-03-playtest-20221203.md [hunk 22]
```diff
@@ -0,0 +1,88 @@
+---
+kind: article
+title: "Playtest 20221203"
+author: "PunkPun"
+created_at: 2022-12-03 00:00 +0000
```
Comment (abcdefg30) [inline news/_posts/2022-12-03-playtest-20221203.md:5]:
```suggestion
created_at: 2022-12-03 01:00 +0200
```

File: news/_posts/2022-12-03-playtest-20221203.md [hunk 23]
```diff
@@ -0,0 +1,88 @@
...
+ * Loaders for more audio formats (e.g., mp3 and ogg), and formats used by the remasters
+ * Many linting and documentation updates
+
+As well as some other changes worth mentioning:
+ * Toolchain and changes to support the .NET 6 runtime
+ * A massive amount of upgrades to both ingame and client UI
+ * Many more hotkeys became rebindable
+ * Substantial progress towards localisation
+
+All in all, 615 pull requests were merged since last release and 552 issues were closed bringing the total number of open tickets down by 141.
```
Comment (abcdefg30) [inline news/_posts/2022-12-03-playtest-20221203.md:86]: Do we need to update this too?

**Review by abcdefg30 — APPROVED** [review]
We'll do the Battlefield news separately.

File: news/_posts/2022-12-03-playtest-20221203.md (hunk 23 above)
Comment (PunkPun) [reply news/_posts/2022-12-03-playtest-20221203.md:86]: I had updated not too long ago. It's about accurate and I can't really make it more accurate. Github tools aren't flexible enough
